community/contribute/README.md

```diff
@@ -15,3 +15,15 @@ To submit an RFP, head to the [RFP forum category](https://forum.pokt.network/c/
 A PEP is a type of proposal that requests a POKT grant from the DAO treasury. You could get a POKT grant for contributions to the Pocket Network ecosystem.
 
 Read more about [submitting a proposal](../governance/proposals.md).
+
+### Submit a PR - Become an Active Pocket V1 Contributor
+
+You can become an open-source contributor by submitting code contributions to the [core Pocket repository](https://github.com/pokt-network/pocket) or the [specifications](https://github.com/pokt-network/pocket-network-protocol).
+
+All the issues are documented and tagged on [Github](https://github.com/pokt-network/pocket/issues), and the [starter task](https://github.com/pokt-network/pocket/issues?q=is%3Aissue+is%3Aopen+label%3A%22starter+task%22) tag can be used to filter for simpler issues to get you acquainted with the codebase.
```

Comment:
Super minor, but I think the word "tag" should be inside the link text (as in, `[starter task tag](...)` or `["starter task" tag](...) `).

Reply:
Done.